### PR TITLE
Get working

### PR DESCRIPTION
**Pull request by somexpert**
ping @ldko, this is ready for review

### CODE REVIEW

**Comment by ldko** [comment]
Should there be initial migrations with this?

File: README.md [hunk 1]
```diff
@@ -32,7 +32,24 @@ Coming soon.
...
 -------
 
-Coming soon.
+To manually run the tests with your current version of Python (system or virtual environment):
+
+1. Install the test Requirements:
+       pip install -r requirements/requirements-test.txt
+
+2. Set the environment variable pointing to the test settings file:
+       export DJANGO_SETTINGS_MODULE=tests.DJANGO_SETTINGS_MODULE
```
Comment (ldko) [inline README.md]:
```suggestion
       export DJANGO_SETTINGS_MODULE=tests.settings
```

**Comment by somexpert** [comment]
Was planning on making the initial migration in the next PR

File: README.md [hunk 2]
```diff
@@ -32,7 +32,24 @@ Coming soon.
 Testing
 -------
 
-Coming soon.
+To manually run the tests with your current version of Python (system or virtual environment):
+
+1. Install the test Requirements:
+       pip install -r requirements/requirements-test.txt
```
Comment (ldko) [inline README.md]:
```suggestion
       pip install -r requirements.txt
```

Comment (somexpert) [reply README.md]: my requirements files got messy there at the end.... when I wrote this, it _did_ install Django, but not anymore

**Pull request merged**